android/src/main/java/com/brentvatne/react/ReactVideoView.java

```diff
@@ -63,9 +63,20 @@ public boolean dispatchKeyEvent(KeyEvent event) {
     }
 }
 
+class CustomScalableVideoView extends ScalableVideoView {
+    public CustomScalableVideoView(Context context) {
+        super(context, null);
+    }
+
+    public int getCurrentPosition() {
```

Comment:
Not sure if it's necessary, but probably should add Override here?
```suggestion
    @Override
    public int getCurrentPosition() {
```